kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/SchemaManagerTest.java

```diff
@@ -377,4 +478,4 @@ private SinkRecord recordWithValueSchema(Schema valueSchema) {
     when(result.valueSchema()).thenReturn(valueSchema);
     return result;
   }
-}
+}
```

Comment:
Unnecessary formatting change; please revert.

Reply:
Formatting changes completed, please verify. Thanks

Reply:
This is still incorrect; see the diff in the GitHub UI.

Reply:
Added the additional line back.